core/ingest/pom.xml

```diff
@@ -92,6 +92,32 @@
 			<artifactId>jsr203hadoop</artifactId>
 			<version>1.0.1</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.hbase</groupId>
+			<artifactId>hbase-client</artifactId>
+			<exclusions>
+				<exclusion>
+					<artifactId>jdk.tools</artifactId>
+					<groupId>jdk.tools</groupId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.apache.hadoop</groupId>
+					<artifactId>hadoop-core</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency> <!-- Spark dependency -->
```

Comment:
we should move these operations to the geowave-analytic-spark project (or we could instead introduce a new geowave-core-spark) which will move these dependencies outside of core-ingest